### ISSUE TITLE
Staging Review finding: Error is not being focused correctly

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#121917]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization - Agile Six Team
**Product:** Marital Status Questionnaire (VA Form 21P-0537)
**Feature:** 21P-0537 Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** Yes
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On Details about your remarriage, when a field is in error, and the user clicks continue focus is moved to the entire form. 

Duplicate of department-of-veterans-affairs/vets-design-system-documentation#4993

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to recreate:
- with Voiceover on, in Safari, navigate to Details about your remarriage.
- Fill in all of the fields, except for How old were you when you remarried?
- TAB to Continue and hit ENTER
- Note behavior
  - VO cursor is on the form `<fieldset>`
  - Programmatic focus is on the first field in error parent container

Other occurences:
- Ocurring for almost all forms on staging.va.gov
- Similar behavior on iOS/VO/Safari

Additional detail:
- Confirmed across multiple forms and with multiple users
- Appears to be a platform/AT issue, specific to how Voiceover is handling groups
- More investigation is needed

<img width="568" height="800" alt="focus mismanaged" src="https://github.com/user-attachments/assets/fb9a5669-4db9-4dc9-9e19-49d03753a371" />
---
https://github.com/user-attachments/assets/b99ca1e2-eb2e-4313-bc1c-75e843cc4b26
---
iOS/VO/Safari
- VO cursor is on the h1, programmatic focus is on the field in error parent container
<img width="1170" height="2532" alt=VO cursor issue" src="https://github.com/user-attachments/assets/8d42cae1-1fe3-439b-aac3-c2b137e722a8" />

</details>

## Recommended action

Ensure focus is moved to the field in error, so that screen reader users and zoomed in users are aware of the field in error. 

https://design.va.gov/templates/forms/accessibility-guidelines#managing-focus-in-form-flows

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 3.3.1 Error Identification - Level A](https://www.w3.org/TR/WCAG22/#error-identification)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** [Forms template](https://design.va.gov/templates/forms/)
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by amponce** [comment]
👋

**Comment by michaelclement** [comment]
Confirmed this is fixed. @natejones6 Closing as resolved.

<img width="766" height="261" alt="Image" src="https://github.com/user-attachments/assets/9b77e7d7-2a80-404b-af49-688e6025331f" />
